### DISCUSSION TITLE
Is there a way to rename or delete a workspace?

### DISCUSSION THREAD

**Discussion by anastk** [AntlerVC/firetable#840] [Support / Q&A] [answered]
I made a mistake and created a workspace in the name of a project thinking that I can create multiple workspaces, now I can't find any way to rename / delete a workspace. 

I need to do this to avoid confusion with team members that I share the workspace with as we are working on two separate projects with different teams and names of course. 

So is there any way to rename a workspace or even delete and start from scratch?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @anastk, Rowy has added the feature of renaming workspaces in a recent release. You can simply access the Workspace settings to customize the workspace and manage its projects.

Attaching a video for reference:

https://user-images.githubusercontent.com/53828745/198605085-3fc40910-b71e-4157-b39b-71d45aa1c647.mp4
